docs/content.zh/docs/deployment/elastic_scaling.md

```diff
@@ -23,134 +23,132 @@ specific language governing permissions and limitations
+在 Apache Flink 中，可以通过手动停止 Job，然后从停止时创建的 Savepoint 恢复，最后重新指定并行度的方式来重新伸缩 Job。
 
-This page describes options where Flink automatically adjusts the parallelism instead.
+这个文档描述的特性是 Flink 如何自动地调整并行度。
 
-## Reactive Mode
+## Reactive 模式
 
 {{< hint info >}}
-Reactive mode is an MVP ("minimum viable product") feature. The Flink community is actively looking for feedback by users through our mailing lists. Please check the limitations listed on this page.
+Reactive 模式是一个 MVP （minimum viable product，最小可行产品）特性。目前 Flink 社区正在积极地从邮件列表中获取用户的使用反馈。请注意文中列举的一些限制。
 {{< /hint >}}
 
-Reactive Mode configures a job so that it always uses all resources available in the cluster. Adding a TaskManager will scale up your job, removing resources will scale it down. Flink will manage the parallelism of the job, always setting it to the highest possible values.
+在 Reactive 模式下，Job 会使用集群中所有的资源。当增加 TaskManager 时，Job 会自动扩容。当删除时，就会自动缩容。Flink 会管理 Job 的并行度，始终会尽可能地使用最大值。
 
-Reactive Mode restarts a job on a rescaling event, restoring it from the latest completed checkpoint. This means that there is no overhead of creating a savepoint (which is needed for manually rescaling a job). Also, the amount of data that is reprocessed after rescaling depends on the checkpointing interval, and the restore time depends on the state size. 
+当发生伸缩时，Job 会被重启，并且会从最新的 Checkpoint 中恢复。这就意味着不需要花费额外的开销去创建 Savepoint。当然，所需要重新处理的数据量取决于 Checkpoint 的间隔时长，而恢复的时间取决于状态的大小。
 
-The Reactive Mode allows Flink users to implement a powerful autoscaling mechanism, by having an external service monitor certain metrics, such as consumer lag, aggregate CPU utilization, throughput or latency. As soon as these metrics are above or below a certain threshold, additional TaskManagers can be added or removed from the Flink cluster. This could be implemented through changing the [replica factor](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#replicas) of a Kubernetes deployment, or an [autoscaling group](https://docs.aws.amazon.com/autoscaling/ec2/userguide/AutoScalingGroup.html) on AWS. This external service only needs to handle the resource allocation and deallocation. Flink will take care of keeping the job running with the resources available.
- 
-### Getting started
+借助 Reactive 模式，Flink 用户可以通过一些外部的监控服务产生的指标，例如：消费延迟、CPU 利用率汇总、吞吐量、延迟等，实现一个强大的自动伸缩机制。当上述的这些指标超出或者低于一定的阈值时，增加或者减少 TaskManager 的数量。在 Kubernetes 中，可以通过改变 Deployment 的[副本数（Replica Factor）](https://kubernetes.io/zh/docs/concepts/workloads/controllers/deployment/#replicas) 实现。而在 AWS 中，可以通过改变 [Auto Scaling 组](https://docs.aws.amazon.com/zh_cn/autoscaling/ec2/userguide/AutoScalingGroup.html) 来实现。这类外部服务只需要负责资源的分配以及回收，而 Flink 则负责在这些资源上运行 Job。
 
-If you just want to try out Reactive Mode, follow these instructions. They assume that you are deploying Flink on a single machine.
+<a name="getting-started"></a>
```

Comment:
Because it is linked by [this line](https://github.com/apache/flink/pull/16928/commits/371d9adf0d21fc8204c414583dcf7994787f872f#diff-69b2dca8ca68dd36524d019124dd2eef1e7914f45f6992733e7d221c13b68ea9R115). Shall I replace this anchor with the Chinese one, e.g. `#入门` instead of `#getting-started`, or shall we follow the recommendation "4. 标题锚点链接" on [this page](https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications)?

Reply:
Thanks for your explanation! WRT to the anchor question, cc @wuchong.

Reply:
Yes, this line is needed if we would like to preserve the original tag links.